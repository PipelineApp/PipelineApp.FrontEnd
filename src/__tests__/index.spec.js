describe('jest verification', () => {
	it('should render valid snapshot', () => {
		const element = { testProp1: 'test', testProp2: 124251234 };
		expect(element).toMatchSnapshot();
	});
	it('should assert correctly', () => {
		expect(1).toBe(1);
	});
});
