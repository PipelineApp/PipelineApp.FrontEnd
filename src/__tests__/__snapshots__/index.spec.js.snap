// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest verification should render valid snapshot 1`] = `
Object {
  "testProp1": "test",
  "testProp2": 124251234,
}
`;
